dev/com.ibm.ws.concurrent.mp_fat/fat/src/com/ibm/ws/concurrent/mp/fat/FATSuite.java

```diff
@@ -19,8 +19,8 @@
                 MPConcurrentTest.class,
                 MPConcurrentConfigTest.class,
                 MPConcurrentJAXRSTest.class,
-                MPConcurrentTxTest.class,
-                MPContextProp1_1_Test.class,
+                //MPConcurrentTxTest.class, //Relies on Jakarta EE 9 but MicroProfile does not yet support EE9
```

Comment:
How to track this so that we can enable this with future MP 5.0 changes. Should an issue be opened for this?

Reply:
https://github.com/OpenLiberty/open-liberty/issues/14941